### ISSUE TITLE
Allow direct navigation to filtered page of task table results using URL parameters

### ISSUE THREAD

**Issue by lowellrex** [department-of-veterans-affairs/caseflow#11055]
#11054 completed the move from paginating, filtering, and sorting on the front-end to doing the same thing using API requests to the back-end. With that foundation we can enable direct navigation to a filtered, ordered page of tasks within a tab. For example, navigating directly to `/organizations/vlj-support-staff?tab=on_hold&page=3&sort_by=case_details_link&order=desc` would load the third page of on hold tasks for the VLJ support staff sorted in descending order by case details link.

## 🚫 Blocked by
* [x] Enable TaskTable to use API requests to paginate, filter, and sort tasks (#11054)

## Acceptance criteria
* [ ] Users can navigate directly to page of task table.

**Comment by lomky** [comment]
@lowellrex We believe this is a duplicate of work we'll have already done in #11052. What do you see being done distinctly in this task?

**Comment by lowellrex** [comment]
> @lowellrex We believe this is a duplicate of work we'll have already done in #11052. What do you see being done distinctly in this task?

This is the front-end work. Navigating `/organizations/vlj-support-staff?tab=on_hold&page=3&sort_by=case_details_link&order=desc` should cause the front-end pagination, filtering, and sorting components to display correct, as well as making the correct API call to the back-end (which might be identical to the URL).

**Comment by lowellrex** [comment]
@hschallhorn #12060 looks great! But do we have a ticket to make the URL in the address bar reflect the current set of tabs, pages, filters, and sorts (it could also serve double duty as the source of truth for that information instead of relying on local state or the redux store)?

In the example below I would expect the `tab` parameter in the URL to change to reflect the current tab. Doing this would allow folks to bookmark a particular set of tabs, pages, filters, and sorts and share the URL with a coworker.

![tab-name-unchanged](https://user-images.githubusercontent.com/32683958/64961510-ac902300-d863-11e9-8bf2-c51faa7ded34.gif)

**Comment by hschallhorn** [comment]
I did not! I actually like that better than creating an entire modal to get the sharable link. updating the address bar seems more intuitive. I'll update that ticket.
